Gems/AWSCore/Code/CMakeLists.txt

```diff
@@ -166,3 +166,8 @@ if(PAL_TRAIT_BUILD_TESTS_SUPPORTED)
         )
     endif()
 endif()
+
+install(DIRECTORY "Tools/ResourceMappingTool"
+    DESTINATION "Gems/AWSCore/Code/Tools"
+    PATTERN "__pycache__" EXCLUDE
+)
```

Comment:
Add:
```
     COMPONENT ${CMAKE_INSTALL_DEFAULT_COMPONENT_NAME}
```
we need to tag all the installed things so the installer picks it up

Reply:
With that change I am good with this, we later will at least wrap it with a method so we can later consolidate all calls
(e.g. we need to force all to define components or pass a default one so the installer picks it up)